src/main/java/edu/kis/powp/jobs2d/command/ICompoundCommand.java

```diff
@@ -13,6 +13,6 @@
 
     @Override
     default void accept(Visitor visitor) {
-        visitor.visitICompoundCommand(this);
+        visitor.visit(this);
```

Comment:
Please do not rename existing code while working on a feature. Such refactoring is not a necessity and should be done as separate commit and PR.